src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -222,11 +222,16 @@ typedef struct uct_ib_mlx5_devx_umem {
 } uct_ib_mlx5_devx_umem_t;
 
 
+/**
+ * LRU cache entry of indirect rkeys
+ */
 typedef struct uct_ib_mlx5_mem_lru_entry {
-    ucs_list_link_t list;
-    void            *indirect_mr;
-    uint32_t        rkey;
-    uint8_t         is_dummy;
+    ucs_list_link_t        list;            /* Entry in the linked list */
+    struct mlx5dv_devx_obj *indirect_mr;    /* Pointer to MR object */
+    uint32_t               rkey;            /* RKey value, also used as a key in
+                                               hash */
+    uint8_t                is_dummy;        /* If true, need to destroy
+                                               @ref indirect_mr on pop */
```

Comment:
/* Whether the associated indirect_mr is created only to prevent reusing the same key (true) or it's referenced by an existing memory handler (false). If true, we need to destroy the indirect_mr when removed from LRU. */